iis-wf/iis-wf-referenceextraction/src/main/java/eu/dnetlib/iis/wf/referenceextraction/community/input/CommunityReferenceExtractionInputTransformerJob.java

```diff
@@ -65,8 +65,11 @@ public static void main(String[] args) throws InterruptedException, IOException
     private static List<Community> convert(Concept concept, String acknowledgementParamName) {
         List<Community> communities = new ArrayList<>();
         for (Param param : concept.getParams()) {
-            if (acknowledgementParamName.equals(param.getName()) && StringUtils.isNotBlank(param.getValue())) {
-                communities.add(buildCommunity(concept.getId(), concept.getLabel(), param.getValue()));
+            //changing the rule by accepting empty ackstmt
+            // if (acknowledgementParamName.equals(param.getName()) && StringUtils.isNotBlank(param.getValue())) {
+            if (acknowledgementParamName.equals(param.getName())) {
+                communities.add(buildCommunity(concept.getId(), concept.getLabel(),
+                        StringUtils.isNotBlank(param.getValue()) ? param.getValue() : ""));
             }
         }
         return communities;
```

Comment:
1. I think `StringUtils.defaultIfBlank(param.getValue(),  "")` does the same thing as `StringUtils.isNotBlank(param.getValue()) ? param.getValue() : ""`
1. IDE complains about using `CharSequence` with `String.equals` here: `cknowledgementParamName.equals(param.getName())`
1. Using streams can make it more readable
```java
        return concept.getParams().stream()
                .filter(x -> acknowledgementParamName.equals(x.getName().toString()))
                .map(x -> buildCommunity(concept.getId(), concept.getLabel(), StringUtils.defaultIfBlank(x.getValue(), "")))
                .collect(Collectors.toList());
```